src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -1844,6 +1844,7 @@
   declare_c2_type(CopySignFNode, Node)                                    \
   declare_c2_type(SignumDNode, Node)                                      \
   declare_c2_type(SignumFNode, Node)                                      \
+  declare_c2_type(IsInfiniteFNode, Node)                                  \
```

Comment:
Where are other new nodes?

Reply:
I did not know how to combine the 3 nodes into one. Could you pls give few pointers to refer to for implementing them?

Reply:
Fixed it and pushed a new update. Sorry, that got missed!

Reply:
Good.